tests/src/drivers/key_management.c

```diff
@@ -154,6 +178,57 @@ psa_status_t test_opaque_export_key(
     const uint8_t *key, size_t key_length,
     uint8_t *data, size_t data_size, size_t *data_length )
 {
+#if defined(MBEDTLS_PSA_CRYPTO_BUILTIN_KEYS)
+    if( psa_key_id_is_builtin( MBEDTLS_SVC_KEY_ID_GET_KEY_ID( psa_get_key_id( attributes ) ) ) )
```

Comment:
The driver should rely only on the key slot number. The key identifier is and should be of no use to the driver.
If you remove this you can remove the guard above as well.